packages/quill/test/unit/core/editor.spec.ts

```diff
@@ -162,6 +166,30 @@ describe('Editor', () => {
         <p><br></p>`);
     });
 
+    test('insert soft line', () => {
```

Comment:
nit pick: could we adjust the naming on all these tests to be "soft line break"?